src/nsidc/metgen/readers/generic.py

```diff
@@ -22,20 +20,13 @@ def extract_metadata(
     It relies on spatial/spo files for geometry information and premet files
     for temporal information.
     """
-    metadata = {
-        "size_in_bytes": os.path.getsize(data_file),
-        "production_date_time": configuration.date_modified,
-    }
+    metadata = {}
 
     # Get temporal information from premet if available
     if premet_content:
-        temporal = utilities.temporal_from_premet(premet_content)
-        if temporal:
-            metadata["temporal"] = temporal
-        else:
-            metadata["temporal"] = [configuration.date_modified]
```

Comment:
Didn't catch this the first time around -- alas, `date_modified` cannot serve as a backup value for the data's temporal coverage.